CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.60.0](https://github.com/serverless/serverless/compare/v1.59.0...v1.60.0) (2019-12-18)
```

Comment:
Shouldn't the diff be against v1.59.3?

Reply:
ah.. good you've noticed. Fast releases as they come from derive branches, are unfortunately not properly detected by `standard-version`.

I'll think whether we can somehow improve that detection

I've updated PR

Reply:
> I'll think whether we can somehow improve that detection

:+1: